core/meta/src/TClass.cxx

```diff
@@ -3451,38 +3451,31 @@ const char *TClass::GetSharedLibs()
 
 TList *TClass::GetListOfBases()
 {
-   if (!fBase) {
-      if (fCanLoadClassInfo) {
```

Comment:
This 'if statement' was removed.  Could you add comments (at the very least in the commit) why it is not needed.